packages/strapi-connector-bookshelf/lib/migrations/index.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const dpMigration = require('./draft-publish-migration');
+
+const migrations = [dpMigration];
```

Comment:
good idea. We could push it a bit further by creating a `migrationRunner` that will wrap those lifecycles logic arroudn the schemaMigration

Reply:
something like that :
```
await migrationRunner(createOrUpdateTables)({ ORM, loadedModel, definition, connection, model });
```
?